.github/workflows/deploy-cpp.yml

```diff
@@ -10,13 +10,18 @@ on:
 jobs:
 
   make_sdist:
-    name: "Make SDist"
+    name: Make SDist
```

Comment:
I started indiscriminately quoting strings in YAML after being burned by some cases that looked like strings but weren't, due to some special character. It never _hurts_ to quote. I don't think it's harder to read with quotes.

This one can be unquoted, of course.